server/app.ts

```diff
@@ -80,7 +80,7 @@ app.use(morgan("hackgt"));
 
 (mongoose as any).Promise = global.Promise;
 mongoose.connect(MONGO_URL, {
-	useMongoClient: true
+	useNewUrlParser: true
```

Comment:
Why is this necessary?

Reply:
I'm not sure if it's necessary, but I did it to get rid of this warning:
```
(node:13240) DeprecationWarning: current URL string parser is deprecated, and will be removed in a future version. To use the new parser, pass option { useNewUrlParser: true } to MongoClient.connect.
```